source/semver.d

```diff
@@ -132,7 +132,7 @@ struct SemVer
     /**
      * Return the canonical string format.
      */
-    string toString() const scope @safe
+    string toString() const @safe
```

Comment:
Why was this removed ? Logically, it seems that `toString` should be `scope`. What prevents it from being `scope` ?

Reply:
You return a string created inside the function scope. The next dmd compiler complains about it, when scope is set. But returning a string is intended here. And the string is stored in the GC, so a 'scope' attribute isn't needed here.

Or the next dmd is wrong here...